e2e/tests/explore/explore.spec.ts

```diff
@@ -1,56 +1,143 @@
-import { test, Page, expect } from '@playwright/test'
-import { Section } from '@shared/dto/sections/section.entity'
-import { E2eTestManager } from '@shared/lib/e2e-test-manager'
+import { test as base, expect } from '@playwright/test';
+import { Section } from '@shared/dto/sections/section.entity';
+import { PageFilter } from '@shared/dto/widgets/page-filter.entity';
+import { E2eTestManager } from '@shared/lib/e2e-test-manager';
 
-test.describe.configure({ mode: 'serial' })
+type TestFixtures = {
+  testManager: E2eTestManager;
+  sections: Section[];
+  requiredFilter: PageFilter;
+};
+
+const test = base.extend<TestFixtures>({
```

Comment:
I remember @andresgnlez and myself explored this approach but somehow we couldn't make it work, I don't even remember what was the goal.

Could you please explain where are you making use of this?

Reply:
Very cool!